Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub.py

```diff
@@ -1475,19 +1475,18 @@ def handle_fetch_results(
     """
     incidents = []
     if pulled_msg_ids and max_publish_time:
-        if last_run_time <= max_publish_time:
```

Comment:
Was it in the original integration or added on a later phase? Can it be related to a duplication bug?

Reply:
It was in the original integration